docs/src/man/basics.md

```diff
@@ -1565,40 +1565,1265 @@ julia> german[Not(5), r"S"]
                 984 rows omitted
 ```
 
-## Basic Usage of Transformation Functions
-
-In DataFrames.jl we have five functions that we can be used to perform
-transformations of columns of a data frame:
-
-- `combine`: creates a new data frame populated with columns that are results of
-  transformation applied to the source data frame columns, potentially combining
-  its rows;
-- `select`: creates a new data frame that has the same number of rows as the
-  source data frame populated with columns that are results of transformations
-  applied to the source data frame columns;
-- `select!`: the same as `select` but updates the passed data frame in place;
-- `transform`: the same as `select` but keeps the columns that were already
-  present in the data frame (note though that these columns can be potentially
-  modified by the transformation passed to `transform`);
-- `transform!`: the same as `transform` but updates the passed data frame in
-  place.
-
-The fundamental ways to specify a transformation are:
-
-- `source_column => transformation => target_column_name`; In this scenario the
-  `source_column` is passed as an argument to `transformation` function and
-  stored in `target_column_name` column.
-- `source_column => transformation`; In this scenario we apply the
-  transformation function to `source_column` and the target column names is
-  automatically generated.
-- `source_column => target_column_name` renames the `source_column` to
-  `target_column_name`.
-- `source_column` just keep the source column as is in the result without any
-  transformation;
-
-These rules are typically called transformation mini-language.
-
-Let us move to the examples of application of these rules
+## Basic Usage of Manipulation Functions
```

Comment:
A larger question - maybe create a separate page for this tutorial?

Reply:
I think that would be better. Maybe "Manipulation Functions" under "User Guide" before "Split-apply-combine"?

Reply:
As commented below - I would put it as a "top level" with a name something along "A gentle introduction to manipulation functions" (so that we clearly signal that this material is less formal than the rest of the manual).

Reply:
What would you think of making the new Top Level section something like "Beginner's Guide" or "User's Guide for Beginners" and then placing "Manipulation Functions" at a second level under that? I'm not volunteering to rewrite the entire User's Guide, but it could leave room for others to add similar "gentle" content to the documentation. It would also make the sidebar look cleaner by splitting up the current long name.

Reply:
Makes sense. The other section that could go there is https://dataframes.juliadata.org/stable/man/basics/ as it has the same objective.

Reply:
Oh, but Basics contains the existing "Basic Usage of Manipulation Functions". I don't know how to differentiate it from this new section if they live next to each other.

I initially intended to just clarify some topics within that section, but now the scope has grown.

I can maybe overwrite that section if I add these topics that I don't currently cover:
> * "Note that this time we use string column selectors because some of the column names have spaces in them."
> * "The benefit of `select` or `combine` over indexing is that it is easier to get the union of several column selectors."
> * "It is important to note that `select` always returns a data frame, even if a single column selected as opposed to indexing syntax."
> * "By default `select` copies columns of a passed source data frame. In order to avoid copying, pass the `copycols=false` keyword argument."

 The other sections under Basics use the German dataset, but I think it is easier to understand what is going on with smaller data frames where you know all the data values.

Reply:
> I can maybe overwrite that section

Yes - I think it is OK just to expand that section (especially that it is top-level now already)

> I think it is easier to understand what is going on with smaller data frames

Agreed. just please use different variable names than these already used there so that using different dataframes does not lead to confusion.

Thank you! (sorry for so many comments, but - unfortunately - writing documentation is hard)

Reply:
Done (sort of). I did not use new variables names though. I had been and continued to just overwrite the definition of `df`. My data frames are so small and frequent that coming up with a new name each time would be a pain.